aries-site/src/examples/templates/forms/SignInExample.js

```diff
@@ -118,6 +123,10 @@ export const SignInExample = () => {
   // eslint-disable-next-line no-unused-vars
   const onSubmit = ({ value, touched }) => {
     // Your submission logic here
+    // For demonstration purposes, your call to API returns a credential error
```

Comment:
```suggestion
    // For demonstration purposes, we are mocking a validation where the password is incorrect.
    // This will cause the error state to appear.
```